changelog/seqera-cloud/v25.3.0_cycle31.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Seqera Cloud v25.3.0_cycle31
+date: 2025-11-12
+tags: [seqera cloud]
+---
+
+## General updates and improvements
+
+- **Studios**
+  - Improved Studio creation responsiveness by preparing Docker build context files asynchronously, allowing scripts and configuration files to be included without blocking the initial request.
+- **Data Explorer**
+  - Added support for LakeFS Cloud in Data Explorer by enabling image preview and file upload functionality when using LakeFS Cloud as an S3-compatible storage provider.
+- Improved job failure handling in AWS Batch compute environments. Improved AWS Batch workflow behavior to automatically terminate unschedulable jobs instead of leaving them pending indefinitely, providing faster feedback when compute resources are insufficient for workflow requirements.
```

Comment:
```suggestion
- Improved AWS Batch workflow behavior to automatically terminate unschedulable jobs instead of leaving them pending indefinitely, providing faster feedback when compute resources are insufficient for workflow requirements.
```
This felt like the same point twice, please ignore if I've read this wrong.